src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -216,97 +325,5459 @@ static ColliderJntSphInit D_80B7A888 = {
+    Player* player = PLAYER;
+    Vec3f zeroVec = { 0.0f, 0.0f, 0.0f };
+    Vec3f spA8;
+    Vec3f sp9C;
+    Vec3f sp90;
+    Input* input = &globalCtx->state.input[0];
+    Vec3f sp80;
+    f32 sp7C;
+    f32 sp78;
+    f32 phi_f0;
+    f32 sp70;
+    Vec3f sp64;
+    Vec3f sp58;
+    s32 pad;
+
+    D_80B7E0AE++;
+
+    if (D_80B7E0B0 != 0) {
+        D_80B7E0B0--;
+    }
+
+    if (D_80B7E0B2 != 0) {
+        D_80B7E0B2--;
+    }
+
+    if (D_80B7E0B4 != 0) {
+        D_80B7E0B4--;
+    }
+
+    if (D_80B7E122 != 0) {
+        D_80B7E122--;
+    }
+
+    if (D_80B7E150 != 0) {
+        D_80B7E150--;
+    }
+
+    if (D_80B7A6A4 != 0) {
+        D_80B7A6A4--;
+    }
+
+    if (D_80B7E0A4 != 0) {
+        D_80B7E0A4--;
+    }
+
+    if (D_80B7E114 != 0) {
+        D_80B7E114--;
+    }
+
+    if (D_80B7E0AC == 1) {
+        D_80B7E0AC = 2;
+        D_80B7E084 = 0;
+        D_80B7E082 = 0;
+        D_80B7E0B6 = 0;
+
+        if (((sLinkAge == 1) && (HIGH_SCORE(HS_FISHING) & 0x400)) ||
+            ((sLinkAge != 1) && (HIGH_SCORE(HS_FISHING) & 0x800))) {
+            sSinkingLureLocation = (u8)Rand_ZeroFloat(3.999f) + 1;
+        }
+
+        D_80B7E148 = 520.0f;
+        D_80B7E144 = 195.0f;
+
+        D_80B7A694 = D_80B7E0B6 = D_80B7E0AE = D_80B7E0B0 = D_80B7E0B2 = D_80B7E0B4 = D_80B7E120 = D_80B7E114 =
+            D_80B7E150 = 0;
+        D_80B7E104 = D_80B7E154 = D_80B7E108 = 0.0f;
+
+        D_80B7E128 = zeroVec;
+
+        for (i = 0; i < LINE_SEG_COUNT; i++) {
+            sReelLinePos[i] = zeroVec;
+            sReelLineRot[i] = zeroVec;
+            sReelLineUnk[i] = zeroVec;
+        }
+    }
+
+    SkinMatrix_Vec3fMtxFMultXYZW(&globalCtx->mf_11D60, &sLurePos, &D_80B7AF94, &sProjectedW);
+
+    if (D_80B7A694 == 0) {
+        Math_ApproachF(&D_80B7E108, -800.0f, 1.0f, 20.0f);
+    } else {
+        Math_ApproachF(&D_80B7E108, 300.0f, 1.0f, 20.0f);
+    }
+
+    switch (D_80B7A694) {
+        case 0:
+            D_80B7FEA0 = 0;
+
+            if (KREG(14) != 0) {
+                KREG(14) = 0;
+                D_80B7E0B6 = 2 - D_80B7E0B6;
+                if (D_80B7E0B6 != 0) {
+                    D_80B7E082 = 0;
+                }
+            }
+
+            Math_ApproachF(&D_80B7E144, 195.0f, 1.0f, 1.0f);
+
+            if (player->stateFlags1 & 0x8000000) {
+                D_80B7E0B4 = 0;
+                player->unk_860 = 0;
+            }
+
+            if (D_80B7E0B4 == 0) {
+                if ((D_80B7E0B0 == 0) && (player->unk_860 == 1)) {
+                    D_80B7E0B4 = 37;
+                    func_80106CCC(globalCtx);
+                }
+            } else {
+                sLureRot.x = sReelLineRot[LINE_SEG_COUNT - 2].x + M_PI;
+                sLureRot.y = sReelLineRot[LINE_SEG_COUNT - 2].y;
+
+                if (D_80B7E0B4 == 18) {
+                    D_80B7A694 = 1;
+                    sLurePos = sRodTipPos;
+                    Matrix_RotateY((player->actor.shape.rot.y / 32768.0f) * M_PI, MTXMODE_NEW);
```

Comment:
```suggestion
                    Matrix_RotateY(BINANG_TO_RAD(player->actor.shape.rot.y), MTXMODE_NEW);
```
also on line `sLureRot.y = (D_80B7E13C / 32768.0f) * M_PI;`

Reply:
See my comment above regarding conversion macros. It works fine here and in a few other places like you pointed out, but some places can't be written with these macros so I'm concerned it will add even more inconsistency throughout the file.